e2e/utils.ts

```diff
@@ -114,10 +123,9 @@ export function invisibilityOf(finder: ElementFinder, timeout = AVERAGE_WAIT_TIM
  * Expect that the given `finder` is displayed/visible (or instead hidden) on the DOM.
  * By default, expect the `finder` to be displayed. Pass `false` as second param to expect it to be hidden.
  * @param finder An element finder.
- * @param {String} [errorMessage] The message to display when the expectation fails.
- * @param {Boolean} [value=true] A boolean indicating if the element should be displayed or not. Default to `true`.
+ * @param
  */
-export async function expectDisplayed(finder: ElementFinder, options?: { elementName: string; shouldBeDisplayed?: boolean }) {
+export async function expectDisplayed(finder: ElementFinder, options: { elementName?: string; shouldBeDisplayed?: boolean } = { elementName: 'Element', shouldBeDisplayed: true }) {
```

Comment:
It's useless to define defaults here considering the next lines. Just use an empty object.

```suggestion
export async function expectDisplayed(finder: ElementFinder, options: { elementName?: string; shouldBeDisplayed?: boolean } = {}) {
```

(Also it would be dangerous defining defaults only this way, considering that providing options with only one of the properties would clear all the other default values.)